docs/contributors/contributors-guide.md

```diff
@@ -0,0 +1,163 @@
+
+## Insights: Connecting Your Repositories
+
+OpenSauced [Insights](../features/insights.md) allows you to see the impact of your contributions. You can use Insights Pages to track repositories' growth, analyze work, and connect with others.
+
+## Why Create an Insights Page
+
+- **Tracking Personal Contributions**: Add repositories that you contributed to and track your contributions.
+- **Building Portfolios and Developing Skills**: Contributing to open source projects can be a great way to develop new skills. Your contributions can also be used as samples for your portfolio. Looking for ways to implement your open source experience in your job search journey? Check out our [job search guide](../job-seekers-guide/introduction.md)! 😄
+- **Identifying Potential Projects to Contribute To**: Finding open source projects to contribute to can be difficult. The Insights feature can help you identify projects to contribute to.
+- **Networking and Community Engagement**: Finding people to work and connect with can be very challenging. Through the feature's Contributors tab, you'll be able to quickly grow your network.
+- **Curating Collections for Interest Groups**: You can curate and share repositories that focus on sectors and programming languages you. For example, @CBID2 created [a page of projects led and founded by women](https://app.opensauced.pizza/pages/CBID2/288/dashboard?range=30).
+
+### Creating a New Insight Page
+
+To connect your repositories, click the "Insights" tab on the dashboard. You will be redirected to a page where you can create your new Insight Page.
+
+There are two ways to add repositories to your Insight Page:
+
+1. **Sync GitHub organization**: Syncing your favorite GitHub organizations is a good idea if you want to keep track of their open source project activities and trends.
+2. **Connect individual repositories**:  Connecting individual repositories is a good idea if you who wants to keep track the contributions of a specific repository or group of repositories. For example, you might want to track of projects you have contributed to or would like to contribute to in the future.
+
+![sync insights gif](../../static/gif/insight-page-demo.gif)
+
+### Using Your Insights Page
+
+Once you have connected your repositories, you will be redirected to your Insight Page. Here, you will see a dashboard with an overview of the repositories and the contributors who have contributed to them.
+
+There are three tabs that provide you with more information on these repositories:
+
+#### Reports
+
+The Reports tab allows paid users to filter for more information about their repositories over the last 30 days and to generate a Downloadable CSV. Filters include: Top 100 Repos, Minimum 5 Contributors, Recent, Most Active, and Spammed. This can be helpful if you're a beginner contributor who is looking for mentors or someone who is looking for potential maintainers for your project.
```

Comment:
- > ... to filter for more information about **their repositories** ...

  This is coming from Maintainers perspective.

- > This can be helpful if you're a beginner contributor who is looking for mentors or someone who is looking for potential maintainers for your project.

  I'm not sure if contributors are looking for potential maintainers.

- You would get better understanding if you try to download the CSV file so you can explain this based on your own experience why it's helpful for contributors.

Reply:
> * > ... to filter for more information about **their repositories** ...
>   
>   
>   This is coming from Maintainers perspective.
> * > This can be helpful if you're a beginner contributor who is looking for mentors or someone who is looking for potential maintainers for your project.
>   
>   
>   I'm not sure if contributors are looking for potential maintainers.
> * You would get better understanding if you try to download the CSV file so you can explain this based on your own experience why it's helpful for contributors.

Good idea @adiati98! :)

Reply:
Hi @adiati98, it seems that the Reports option is limited to Pro users, so I can't download the CSV file.

Reply:
@CBID2 CSV file is not sharable on GH. So, I provide the screen recording instead.

**Download Report**

![opensauced report1](https://github.com/open-sauced/docs/assets/45172775/0dafa50c-1709-4e0c-92a6-4870ec23fe2e)


**CSV preview**

![csv report](https://github.com/open-sauced/docs/assets/45172775/f576c359-a6d0-4e8d-845d-02ad69308080)

Reply:
Thanks @adiati98. So, I can say something along the lines of “paid users can use the CSV file to track their progress”?

Reply:
@CBID2 the reason I suggested you to try out the "Report" feature and download the CSV is for you to understand it better so you can tell the benefits of this feature for contributors.

I really like what you said:

> I came up with this idea based on my own experience of being a maintainer and contributor for at LinksHub. It's an interesting place to be in.

As a contributor, I'm really interested to know more about why this feature is an interesting place to be in. 😊

Reply:
@adiati98, when balancing between the maintainer and contributor, sometimes the viewpoints blur. You have to consciously remind yourself what it is like to be new to open source. Lol, that's the best way I can explain it. :)

Reply:
@CBID2 thanks for the changes!

Sorry that I can only make a GIF of screen recording and it can't be paused.

But if you watch it several times, what do you think of it? What can you track there as a contributor?

Reply:
Hi @adiati98. Sorry for the late response. If I had access to this feature as a contributor, I’d use the CSV file in these two ways:

- Networking: Finding people to collaborate with for future contributions 
- Learning New Skills: I’d also copy the link to a specific pull request to see how that person filled out the project’s form and ultimately, their commits so I can see how they went about solving an issue and apply it into my own approach to contributing to open source projects.